entry.inline.js

```diff
@@ -12,3 +12,35 @@ try {
 } catch (error) {
   console.warn("Unable to set nop", error);
 }
+
+try {
+  if (localStorage.getItem("baseline-indicator") === "open") {
```

Comment:
Nit: Not sure it makes sense to wrap everything, would rather just try-catch the localStorage access.

```js
function isOpen() {
  try {
    return localStorage.getItem("baseline-indicator") === "open";
	} catch {
    return false;
  }
}
```